### ISSUE TITLE
NTR: Small intestine biopsy: muscular layer thin (Lab: pathology: biopsies & morphology)

### ISSUE THREAD

**Issue by pnrobinson** [obophenotype/human-phenotype-ontology#9358]
New term request Small intestine biopsy: muscular layer thin (elthat_140726070536): Bowel; intestinal  UMLS: 4011711

**Comment by pnrobinson** [comment]
@MickeySegal
To my knowledge the lamina muscularis mucosae of the small intestine is a relatively thin structure normally. Can you provide more context about how this term would be used and for what diseases?

**Comment by MickeySegal** [comment]
https://www.omim.org/clinicalSynopsis/155310 "Thinning of outer muscular layer of small bowel"

So the finding should include the word "outer".

**Comment by pnrobinson** [comment]
@MickeySegal 
Adding new term: 
- ID: HP:6000072
- label: Thinning of outer muscular layer of small bowel
- parents: [Abnormal small intestine morphology (HP:0002244)](https://hpo.jax.org/app/browse/term/HP:0002244)
- pmid: PMID:717927
- def: The depth of the outer muscular layer of the small bowel is below the lower limit of normal.
- comment: 
- orcid: orcid:0000-0002-0736-9199
- github ID: https://github.com/obophenotype/human-phenotype-ontology/issues/9358
- synonyms:
